InfiniteRecharge-2020.iml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this file has the dumb

delete and disable in settings -> build tools -> gradle -> something about iml files for imported gradle projects